pubspec.yaml

```diff
@@ -1,9 +1,9 @@
 name: os_detect
-version: 1.0.0
+version: 1.1.0-nullsafety.0
```

Comment:
Even though there is only 1 package depending on this we may as well model the migration guid and do a major version bump.

```yaml
version: 2.0.0-nullsafety.0
```